### ISSUE TITLE
Vengeance of Excalibur (1991)

### ISSUE THREAD

**Issue by Python-Exoproject** [exoscoriae/eXoDOS#63]
Can transfer characters from Spirit of Excalibur but it uses a difficult to implement method.

**Comment by Python-Exoproject** [comment]
Notes from discord on subject:

Python17/04/2019
Vengeance of Excalibur:

@eXo Transfer will be interesting to setup. The game handles transfer via the installer as you noted. 

It asks for your floppy drive letter to locate the .sav file from Spirit of Excalibur. It follows this asking if you prefer to load the save from the drive letter you gave, saying "No" makes it look in the game folder. 

Once the .sav has been found and selected it lists all available knights inside that .sav where you get to select only four to transfer. Once that is done the process is complete, next time you start a new game you will have the games prebuilt knights and your transferred four to select from.
I cant see a way to do it outside of the installer as it seems to inject the selected 4 characters into "episode1.dat". If you do it again it replaces the 4 you previously imported keeping the prebuilt 7 intact.

**Comment by Python-Exoproject** [comment]
It looks like at some point the save transfer feature has actually been implemented, albeit via the game installer. It would be nice to have a way to do it without going into the installer but I dont think thats possible,
